runtime/ops/indexing.cc

```diff
@@ -190,6 +191,38 @@ chainerx::Array GatherOp::RunImpl(ChxVMState* st, const chainerx::Array& data, c
     return data.Take(indices, axis);
 }
 
+// TODO(take-cheeze): Move to ChainerX
+chainerx::Array ScatterOp::RunImpl(
+        ChxVMState* st, const chainerx::Array& data_, const chainerx::Array& indices_, const chainerx::Array& updates_) {
+    const int64_t axis = this->axis < 0 ? data_.shape().size() - this->axis : this->axis;
```

Comment:
Sorry, now I remembered what we have. I think you can use `BlitArray`. It should be as simple as like
```
slices = make_slice(indices_);
BlitArray(data_, updates_.At(slices));
```